uber/models/promo_code.py

```diff
@@ -464,7 +464,7 @@ def _attribute_adjustments(self):
             self.uses_allowed = None
 
         # If 'discount' is empty, then this is a full discount, free badge
-        if self.discount == '':
```

Comment:
We actually want a discount of 0 to be possible on promo codes because we use non-discount promo codes to track something in the art show plugin... I should make a subclass for that but I haven't had time.

Reply:
Ok, I think there was a test assuming the opposite, but the test is wrong!

Reply:
That makes sense, probably by the time I changed this code we weren't updating tests. We can comment out the test for now with a note that we can re-enable it once I finally fix Art Show to not use the PromoCode class directly!